static-assets/scripts/main.js

```diff
@@ -536,11 +536,15 @@
                 });
             }
 
-            this.setCookie = function(cookieGenName, value){
+            this.setCookie = function(cookieGenName, value, maxAge){
                 //$cookies[cookieName] = site.siteId;
                 var domainVal;
                 domainVal = (document.location.hostname.indexOf(".") > -1) ? "domain=" + document.location.hostname : "";
-                document.cookie =
+                if (maxAge && maxAge > 0)
```

Comment:
- Please use curly braces despite being a single line
- Use commas instead of + (appreciate if you also change the + of the original line too)
- 0 or negative numbers have a meaning for the `max-age` directive; please change to `maxAge != null` (single equals)